packages/payment-streams/src/index.js

```diff
@@ -29,17 +31,15 @@ const createPaymentStreams = function (web3, options = {}) {
     .getChainId()
     // .then((chainId) => (chainId === 1337 ? 1 : chainId)) // Ganache hack
     .then(function (chainId) {
-      const contract = contracts.PaymentStreamFactory.find(
-        c => c.chainId === chainId
-      )
-      if (!contract) {
+      const constractAddress = options.address || DEFAULT_ADDRESS
+      if (!constractAddress) {
```

Comment:
Minor spelling mistake

```suggestion
      const contractAddress = options.address || DEFAULT_ADDRESS
      if (!contractAddress) {
```

and this is also a few line below

Reply:
FIxed here: Removed the default here: https://github.com/hemilabs/pure.finance/pull/2/commits/50613c94ab27cb3cc111f820f6ba587e14cf0d97